src/js/node/crypto.js

```diff
@@ -1327,6 +1367,12 @@ var require_browser3 = __commonJS({
     var sha = require_sha2();
     var ZEROS = Buffer2.alloc(128);
     function Hmac(alg, key) {
+      if (key instanceof KeyObject) {
+        key = key.export();
+      } else if (key instanceof CryptoKey) {
+        key = KeyObject.from(key).export();
```

Comment:
maybe we should make `export` a static method that does this as one function instead of two

but we can do that in a future PR

Reply:
Fixed on https://github.com/oven-sh/bun/pull/5940/commits/59ebd2d62bf1da38db66be1cb3e00fcd2c56d302